test-it/testList.json

```diff
@@ -31,7 +31,8 @@
         },
         "manual": {
             "tests": [
-                "StringFormatTime.via"
+                "StringFormatTime.via",
+                "TestVI_LargeBDTest.via"
```

Comment:
Could you please try to include the fix but exclude the test "TestVI_LargeBDTest.via" and see whether/how the tests fail.